roles/setup_pgd/tasks/pg_hba_config.yml

```diff
@@ -57,6 +57,64 @@
   run_once: true
   no_log: "{{ disable_logging }}"
 
+- name: Prepare hba non ssl list for user - pg_owner
+  ansible.builtin.set_fact:
+    pg_allow_ip_addresses: >-
+      {{ pg_allow_ip_addresses | default([]) + [
+        {
+          "contype": "host",
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": "postgres"
+          },
+        {
+          "contype": "host",
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": "postgres"
+          },
+          {
+           "contype": "host",
+           "users": pg_owner,
+           "source": node.private_ip + "/32",
+           "databases": "postgres"
```

Comment:
if we are using ident auth, should the method be `ident`?

Reply:
This piece of code was already there. Will update it.